lib/auth/oauth/__tests__/oauth.test.ts

```diff
@@ -260,7 +273,9 @@ describe('callback', () => {
     queueMockResponse(JSON.stringify(successResponse));
 
     const callbackResponse = await shopify.auth.callback({rawRequest: request});
-
+    if (!callbackResponse) {
+      fail('Callback response is undefined');
+    }
```

Comment:
I presume all these callbackResponse checks are unnecessary now that you've removed the possibility of an `undefined` return?